acceptance/acceptance.go

```diff
@@ -68,8 +67,7 @@ func Configure(cfg Configuration) error {
 	planFeatures = cfg.PlanFeatures
 	newPlan = cfg.NewPlan
 	newPlanFeatures = cfg.NewPlanFeatures
-	credentialType = "single"
-	credentialRotationType = "none"
+	credentialType = "none"
```

Comment:
until we introduce the command line flags no rotation or multiple credential tests is performed.